assets/styles/landing.scss

```diff
@@ -0,0 +1,33 @@
+.body {
+    border-top: 8px solid $cream-can;
+    font-family: 'Open Sans';
+    margin: 0;
+}
+
+.body .principal {
```

Comment:
also `principal` is the school's director, what you want is `main` for example :P